pkg/sql/schemachanger/scplan/internal/rules/helpers.go

```diff
@@ -255,6 +255,14 @@ func isColumn(e scpb.Element) bool {
 	return false
 }
 
+func isCheckConstraint(e scpb.Element) bool {
+	switch e.(type) {
+	case *scpb.CheckConstraint:
+		return true
+	}
+	return false
+}
```

Comment:
This function is only ever going to map to one element type so I don't think it's very useful.